handler/home.go

```diff
@@ -17,8 +17,8 @@ const internalErrFormat = `
 `
 
 func (h *Handler) Home(rw http.ResponseWriter, r *http.Request) {
-	err := asset.IndexTemplate.Execute(rw, nil)
-	if err != nil {
+	rw.Header().Set("content-type", "text/html")
+	if _, err := rw.Write(asset.IndexHTML); err != nil {
 		log.Error().Err(err).Msg("unable to execute index.html template")
 		api.WriteHTMLErrorf(rw, http.StatusInternalServerError, internalErrFormat, http.StatusInternalServerError, "Unable to process the request, try again later.")
```

Comment:
This doesn't make sense anymore I guess, when the writing to the response on line 21 fails then writing the error to the response writer will also fail. We can remove all code related to html errors.

Removing this line will also fix the linter for this PR.